packages/newspringchurchapp/ios/newspringchurchapp.xcodeproj/project.pbxproj

```diff
@@ -298,6 +298,27 @@
 			remoteGlobalIDString = 3D3CD9321DE5FBEE00167DC4;
 			remoteInfo = "cxxreact-tvOS";
 		};
+		425A6CCB22779BF60029BA55 /* PBXContainerItemProxy */ = {
+			isa = PBXContainerItemProxy;
+			containerPortal = 3CFDD4BA5D0E4E2AB58B9F7E /* RNReanimated.xcodeproj */;
+			proxyType = 2;
+			remoteGlobalIDString = 134814201AA4EA6300B7C361;
+			remoteInfo = RNReanimated;
```

Comment:
These items were already added in a prior PR. Not sure why they ended up being changed in here?